scripts/lib/lib.js

```diff
@@ -0,0 +1,25 @@
+const {execSync} = require('child_process');
```

Comment:
Most of this code was moved from the CLI.  See react-native-community/cli/pull/2422 for the matching change in CLI, where these were removed.